src/content/blog/2024/special_interest_groups.mdx

```diff
@@ -0,0 +1,155 @@
+in a process much like proposing a new group.
+
+## Who can join?
+
+Anyone can join any interest group.
+Just as every nf-core pipeline is owned by the community, special interest groups
+will be made as inclusive as possible.
+
+We will try to avoid naming groups after existing groups or consortia, though
+many may begin from such origins. We will also try to not to tie any interest
+groups to specific topics or research groups where possible.
+We want anyone working in a relevant field or interest to be able to join (and leave!)
+an interest group without feeling like an outsider.
+
+## What will they do?
+
+We fully expect every interest group to be different: both in their scope and
+membership, but also in the time that their organisers can commit and the activities
+that would be most useful to their members.
+As such, we want to be as flexible as possible with our expectactions.
+If the group is active and people are finding it useful, we will consider it a success.
+
+That said, the types of things that we hope that interest groups could do could include:
+
+-   Writing best practices for nf-core pipeline usage, specific to their field
+-   Creating pipeline configs specific to the group, to encode these recommendations
+-   Work together to harmonise usage of data formats
+-   Listing relevant externala and internal resources, videos and documentation
+-   Holding regular group meetings
+-   Presenting updates in nf-core/bytesize talks and blog posts
+-   Potentially even running their own series of talks, if there is interest
+
+We're sure that our new groups can and will do many more different and diverse things.
+We can't wait to see where the program goes!
+
+## What about pipeline development?
+
+Discussion within special interest groups can revolve around usage of pipelines,
+but when it comes to any missing features or requirements we expect members to move
+discussions into the regular pipeline channels.
+There should be a distinction between scientific and technical topics.
+
+In this way, we hope that interest groups and pipeline groups work in an
+orthogonal manner, each contributing to the other.
+
+<Image src={crossFunctional} alt="Cross-functional pipelines and interest group structure" />
+
+# Introducing: Our first special interest group
+
+Today we are delighted to announce our first interest group: _Domestic animal genomics_.
+We'll leave the details to be elaborated on in a dedicated blog post by them at a later
+date, but we'd like to take this opportunity to thank them for taking the lead.
+
+This interest group has formed out of the [BovReg](https://bovreg.eu/) and
+[EuroFAANG](https://eurofaang.eu/) research consortia.
+BovReg has been working closely with nf-core since its inception and has helped
+bring other EuroFAANG consortia members together to use nf-core pipelines,
+including GEroNIMO, AQUA-FAANG, GENE-SWitCH, Rumigen and HoloRuminant.
```

Comment:
I added the links to the projects and ordered them alphabetical to avoid any 😡 
```suggestion
including [AQUA-FAANG](https://www.aqua-faang.eu/), [EuroFAANG Research Infrastructure](https://eurofaang.eu/), [GENE-SWitCH](https://www.gene-switch.eu/), [GEroNIMO](https://www.geronimo-h2020.eu/), [HoloRuminant](https://holoruminant.eu/) and [Rumigen](https://rumigen.eu/).
```